perun-engine/src/test/java/cz/metacentrum/perun/engine/AbstractEngineTest.java

```diff
@@ -63,9 +64,9 @@ public void setup() throws Exception {
 		engineId = Integer.parseInt(propertiesBean.getProperty("engine.unique.id"));
 
 		// create session
-
+		User user = perun.getUsersManagerBl().getUserById(null, 1);
```

Comment:
I'm not fond with this change. In-memory version of tests might work, since we added SQL to create initial user (perun@BA), but when you run this against real DB, tests might fail, since you can't be sure, there is a User with ID=1. Maybe it would be better to modify methods, which caused a lot of exceptions in logs - createFaciliy() ? We might use simple check, if user is not null in session and add admin only then. I'm not sure, if it caused unnecessary exceptions in other methods.